dash/dash.py

```diff
@@ -125,7 +125,8 @@
             html.Div(id=_ID_DUMMY, disable_n_clicks=True),
         ]
     )
-except AttributeError:
+# pylint: disable=bare-except
+except:  # noqa: E722
```

Comment:
What prompted this change? Also if we do need this, can we do `pylint: disable-next=bare-except` so it doesn't continue to apply to the rest of the file?

Reply:
I had built the component on an older branch, when I got back to dev it couldn't build it was not an AttributeError but a TypeError.